Jenkinsfile

```diff
@@ -1,2 +1,5 @@
 // Builds a module using https://github.com/jenkins-infra/pipeline-library
-buildPlugin(platforms: ['linux'])
\ No newline at end of file
+buildPlugin(useContainerAgent: true, configurations: [
+  [platform: 'linux', jdk: 21],
+  [platform: 'windows', jdk: 17],
+])
```

Comment:
Switch from building on Java 8 to building on Java 17 and 21, per the recommended configuration in the archetype.